MapboxGLAndroidSDK/build.gradle

```diff
@@ -67,7 +67,7 @@ android {
 
     buildTypes {
         debug {
-            testCoverageEnabled true
+            //testCoverageEnabled true
```

Comment:
Guess we could remove it all, if I understood @tobrun correctly

Reply:
That sounds good.